Recent cmake seems not to populate INTERFACE_INCLUDE_DIRECTORIES when using target_include_directories with empty path list.

Drop duplicate "decaf" from includes path.

--- libdecaf-1.0.0/cmake/DecafConfig.cmake.in.orig	2018-06-06 00:25:08.000000000 +0200
+++ libdecaf-1.0.0/cmake/DecafConfig.cmake.in	2020-09-26 09:22:59.720744590 +0200
@@ -42,9 +42,7 @@
 		endif()
 	endif()
 endif()
-get_target_property(DECAF_INCLUDE_DIRS ${DECAF_TARGETNAME} INTERFACE_INCLUDE_DIRECTORIES)
-list(INSERT DECAF_INCLUDE_DIRS 0 "@CMAKE_INSTALL_FULL_INCLUDEDIR@")
-list(REMOVE_DUPLICATES DECAF_INCLUDE_DIRS)
+set(DECAF_INCLUDE_DIRS "@CMAKE_INSTALL_FULL_INCLUDEDIR@")
 
 set(DECAF_CPPFLAGS @DECAF_CPPFLAGS@)
 set(DECAF_FOUND 1)
--- libdecaf-1.0.0/src/CMakeLists.txt.orig	2018-06-06 00:25:08.000000000 +0200
+++ libdecaf-1.0.0/src/CMakeLists.txt	2020-09-26 09:39:00.252207612 +0200
@@ -119,7 +119,7 @@
 endif()
 
 install(DIRECTORY ${GSOURCE_PATH}/include/
-	DESTINATION include/decaf
+	DESTINATION include
 	FILES_MATCHING PATTERN "*.h*"
 	PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ
 )
